stack-graphs/src/stitching.rs

```diff
@@ -707,6 +707,13 @@ impl<'a> Display for DisplaySymbolStackKey<'a> {
 //-------------------------------------------------------------------------------------------------
 // Stitching partial paths together
 
+/// Configuration for partial path stitchers.
+#[derive(Clone, Debug, Default)]
```

Comment:
This needs `#[repr(C)]` if you're going to use the `std::mem::transmute` trick in the C bindings